client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -159,20 +155,25 @@ function CheckoutSummaryFeaturesList( props: {
 		( product ) => isDomainProduct( product ) || isDomainTransfer( product )
 	);
 	const hasPlanInCart = responseCart.products.some( ( product ) => isPlan( product ) );
-	const translate = useTranslate();
-	const isJetpackNotAtomic = useSelector( ( state ) =>
-		siteId ? isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId ) : undefined
+	const refundableProducts = responseCart.products.filter(
+		( product ) =>
+			product.cost &&
+			( isDomainProduct( product ) ||
+				isDomainTransfer( product ) ||
+				isPlan( product ) ||
+				isGoogleWorkspace( product ) ||
+				isTitanMail( product ) )
 	);
-
-	const showRefundText = responseCart.total_cost > 0;
-
-	let refundDays = 0;
-	if ( hasDomainsInCart && ! hasPlanInCart ) {
-		refundDays = 4;
-	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
-		refundDays = hasMonthlyPlanInCart ? 7 : 14;
-	}
-	const refundText = getRefundText( refundDays, null, translate );
+	const uniqueRefundableProducts = refundableProducts.reduce< ResponseCartProduct[] >(
+		( acc, p ) => {
```

Comment:
I was intentionally succinct here to avoid unnecessary line wrapping, but I'll revert this in order to stick to our common practices 👍 And in any case, maybe `acc` is quite cryptic when compared to something like `result`.